packages/metadata/src/metadata-service.ts

```diff
@@ -229,7 +229,7 @@ export class MetadataService implements IMetadataService {
     /**
      * Traverses all metadata elements and calls visitor function for each element.
      *
-     * @param visitElement
+     * @param visitElement metadata element
```

Comment:
Misleading description. Parameter is a visitor callback function, not a metadata element.

Reply:
updated